static/app/components/events/eventTagsAndScreenshot/screenshot/modal.tsx

```diff
@@ -103,19 +105,65 @@ function Modal({
     }
   };
 
+  const path = location.pathname;
+  const query = location.query;
   const {dateCreated, size, mimetype} = currentEventAttachment;
   const links = pageLinks ? parseLinkHeader(pageLinks) : undefined;
-  const previousDisabled =
-    links?.previous?.results === false && currentAttachmentIndex === 0;
-  const nextDisabled =
-    links?.next?.results === false &&
-    currentAttachmentIndex === MAX_SCREENSHOTS_PER_PAGE - 1;
+
+  // Pagination behaviour is different between the attachments tab with page links
+  // vs the issue details page where we have all of the screenshots fetched already
+  const paginationProps: ComponentProps<typeof ScreenshotPagination> | null = links
```

Comment:
nit: can we make this an if/else block so its more readable?

Reply:
https://github.com/getsentry/sentry/pull/41337/commits/0aa8a5a555957ce6866a687653d7e5dbc199b687

Done 👍